frontend/src/components/BackButton.tsx

```diff
@@ -0,0 +1,16 @@
+
+
+const BackButton = () => 
+
+{
```

Comment:
change it to the "function" declaration to keep it consistent with the rest of our code